server.js

```diff
@@ -7,7 +7,7 @@ const app = express();
 const bodyParser = require("body-parser");
 const session = require("express-session");
 const SQLiteStore = require("connect-sqlite3")(session);
-const campaigns = require("./campaigns");
+const campaigns = require("./src/campaigns");
```

Comment:
not needed, is it?

Reply:
It's needed. I use it for publishing the campaings coupons to each created customer on line 73.